### PR TITLE
[FIVE-125] (subTicket FIVE-221) Completar UnitTests del controller de categorías

### PR DESCRIPTION
**Pull request by martinmena**
## Tareas realizadas
Se crearon 6 nuevos métodos en CategoriesControllerTests:

- GetAllAsync_ReturnsOk
- SaveAsync_ReturnsOk
- UpdateAsync_ReturnsOk
- UpdateAsync_ReturnsNotFound
- DeleteAsync_ReturnsNoContent
- DeleteAsync_ReturnsNotFound

### CODE REVIEW

**Review by matiasbeckerle — APPROVED** [review]

File: test/FRF.Web.Tests/Controllers/CategoriesControllerTests.cs [hunk 1]
```diff
@@ -67,5 +114,134 @@ public async Task GetAsync_ReturnsNotFound()
...
+
+        [Fact]
+        public async Task DeleteAsync_ReturnsNotFound()
+        {
+            // Act
+            var result = await _classUnderTest.DeleteAsync(1);
+
+            // Assert
+            Assert.IsType<NotFoundResult>(result);
+            _categoriesService.Verify(mock => mock.DeleteAsync(It.IsAny<int>()), Times.Never);
```
Comment (matiasbeckerle) [inline test/FRF.Web.Tests/Controllers/CategoriesControllerTests.cs:244]: I don't remember the delete code but based on this test I'm inferring you have some sort of GET call before the delete one? Anything else to assert?

Comment (martinmena) [reply test/FRF.Web.Tests/Controllers/CategoriesControllerTests.cs:244]: Nope, that's pretty much it. First there's a Get, and if the response is null, the controller returns a NotFound, but if not, executes the DeleteAsync from the service and returns a NoContent (no matter what the response from the service is)

Comment (matiasbeckerle) [reply test/FRF.Web.Tests/Controllers/CategoriesControllerTests.cs:244]: Then can you add a _verify_ -once- for the get call?

Comment (martinmena) [reply test/FRF.Web.Tests/Controllers/CategoriesControllerTests.cs:247]: Done

**Review by gperelstein — APPROVED** [review]
Everything looks excellent!!

**Pull request merged**